net/dns/resolver/tsdns_test.go

```diff
@@ -67,6 +71,16 @@ func unpackResponse(payload []byte) (dnsResponse, error) {
 		return response, nil
 	}
 
+	response.truncated = h.Truncated
+	if response.truncated {
+		// TODO(adrian): Ideally, answer processing should still succeed when
```

Comment:
Mild preference: make TODO reference a bug number rather than a person, so there's continuity if you end up having to go do something else.

Reply:
> can you get a verb into the commit message / PR title, after the colon? (we mostly follow https://golang.org/wiki/CommitMessage)

Ah, I was trying to follow the "standard" 50 char limit for commit subject but combined with the pathname does not leave much room. I see that's not actually a requirement.